spec/integration/machine_spec.rb

```diff
@@ -75,6 +75,157 @@
         ).to eq("ZWNobyAnZm9vJw==\n")
       end
 
+      it "respects the network_interfaces block with maximum attributes", :super_slow do
+        private_ip_address_start = Random.rand(30)+10
```

Comment:
Not really - I have had collision issues before using low IP numbers so I just jumped it over 10